### ISSUE TITLE
Ubuntu 21.04 (hirsute) missing in Ubuntu.ipxe

### ISSUE THREAD

**Issue by Kipjr** [netbootxyz/netboot.xyz#883]
**Describe the bug**
After choosing Ubuntu / Ubuntu Hirsute / Install , I get an error with a very strange url (http://archive.ubuntu.com/ubuntu-installer/amd64/linux). Looks like some variables were never initialized. Got a workaround for this, and the fix looks rather simple. 

It seems that there might be another issue involved regarding a missing 'goto' statement which causes this specific erroneous URL: https://github.com/netbootxyz/netboot.xyz/blob/8bae5e5cb90eba1c1c8e9f308fec623c5932d311/roles/netbootxyz/templates/menu/ubuntu.ipxe.j2#L71
https://github.com/netbootxyz/netboot.xyz/blob/8bae5e5cb90eba1c1c8e9f308fec623c5932d311/roles/netbootxyz/templates/menu/ubuntu.ipxe.j2#L72

**To Reproduce**
1. Choose Linux Network Installs / Ubuntu / Hirsute
2. See error (especially 'ubuntu-installer')

**Expected behavior**
The installation script with some output. 

**Screenshots**

![image](https://user-images.githubusercontent.com/12066560/115998765-1d272600-a5e9-11eb-9fe9-e4f0ed1d0753.png)
![image](https://user-images.githubusercontent.com/12066560/115998777-26b08e00-a5e9-11eb-9b6e-deeded07f972.png)
![image](https://user-images.githubusercontent.com/12066560/115998737-fb2da380-a5e8-11eb-9f1f-2dbce0854c14.png)

**Additional context**

Webapp Version: | 0.6.3
Menus Version: | 2.0.36

**Workaround**
edit ubuntu.ipxe and insert after 
```
:groovy
set kernel_url ${live_endpoint}/ubuntu-squash/releases/download/20.10-424a54b4/
set codename groovy
set version_number 20.10
goto sub_boot
```
this new text
```
:hirsute
set kernel_url ${live_endpoint}/ubuntu-squash/releases/download/21.04-2cb0e384/
set codename hirsute
set version_number 21.04
goto sub_boot

```

**Comment by antonym** [comment]
Can you try the development endpoint via the Utility menu and see if you are still seeing the same result?  That should hopefully be all fixed in the next release.

**Comment by Kipjr** [comment]
![image](https://user-images.githubusercontent.com/12066560/116002914-21a90a00-a5fc-11eb-9ae6-3114eca07d32.png)
It seems to be working.

**Comment by antonym** [comment]
Fixes should be released in 2.0.37 which is rolling out right now.
